src/lib.rs

```diff
@@ -5,6 +5,7 @@
 //! Most functionality can be accessed by creating an instance of [`LightningNode`] and using its methods.
 
 #![allow(clippy::let_unit_value)]
+#![allow(clippy::empty_line_after_doc_comments)]
```

Comment:
Either we do this, or the [already merged change](https://github.com/getlipa/lipa-lightning-lib/pull/1284), but better not both

Reply:
Let's use my solution. Changing the CI toolchain version does not fix the issue on local environments. 

I've added a comment explaining why this is done.